temporalcli/commandsgen/parse.go

```diff
@@ -44,6 +44,14 @@ type (
 		IgnoreMissingEnv       bool     `yaml:"ignores-missing-env"`
 		Options                []Option `yaml:"options"`
 		OptionSets             []string `yaml:"option-sets"`
+		Docs                   Docs     `yaml:"docs"`
+	}
+
+	// Docs represents docs-only information that is not used in CLI generation.
+	Docs struct {
+		// Tags are the same as Keywords, but with `-` instead of ` `.
+		Keywords    []string `yaml:"keywords-tags"`
```

Comment:
```suggestion
		KeywordsTags    []string `yaml:"keywords-tags"`
```

Ideally keep the field names the same as the YAML names (can probably just call these "keywords", that you happen to also use them as tags is a generator detail probably)